### ISSUE TITLE
An exception occurred while training about finding NaN or Inf in the input tensor.

### ISSUE THREAD

**Issue by wangxiyuan9** [jytime/Deep-SfM-Revisited#14]
Best ,when I using 2011_09_26/2011_09_26_drive_0002_sync for training, I found that the loss_depth corresponding to line 392 of the main function will appear NAN. I also suspect that the learning rate is too large. Even if I set it to --lr =0, the following picture will still appear. , please give me some advice if you know how to slove , thanks
![2022-05-12 21-03-30 的屏幕截图](https://user-images.githubusercontent.com/84949506/168081522-e760153b-9946-417f-94a4-152bbf4decd3.png)

**Comment by jytime** [comment]
Hi,

Would you mind checking the batch size of the training? Also, it would be helpful if you could print the network outputs layer by layer to find where the NaN comes from.
